test-helpers/src/docker_compose.rs

```diff
@@ -106,6 +106,17 @@ impl DockerCompose {
         .unwrap();
     }
 
+    /// Restarts the container with the provided service name
+    pub fn start_service(&self, service_name: &str) {
+        run_command(
+            "docker-compose",
+            &["-f", &self.file_path, "start", service_name],
+        )
+        .unwrap();
+
+        // TODO: call wait_for_containers_to_startup
```

Comment:
What's this TODO?

Reply:
I am purposefully leaving it undone.
it needs https://github.com/shotover/shotover-proxy/pull/917 to land before it can be done.
And it turns out I didnt need it for this PR because the test logic has its own delay (receiving the cassandra UP event)
But I think the start_service method should eventually still have its own in built delay.